### PR TITLE
42 sync assembly nmdc a

### PR DESCRIPTION
**Pull request by mflynn-lanl**

### CODE REVIEW

File: webapp/server/crons/pipelineMonitor.js
Comment (yxu-lanl) [inline webapp/server/crons/pipelineMonitor.js:1]: Don't hardcode the options path, get it from the config/workflow.js.
const tmpl = path.join(config.WORKFLOWS.TEMPLATE_DIR, 'metagenome_pipeline_options.json');

File: webapp/server/crons/workflowMonitor.js
Comment (yxu-lanl) [inline webapp/server/crons/workflowMonitor.js:1]: Don't hardcode the options path, get it from the config/workflow.js.
const tmpl = path.join(config.WORKFLOWS.TEMPLATE_DIR, 'metaG_options.tmpl');

File: webapp/server/util/workflow.js
Comment (yxu-lanl) [inline webapp/server/util/workflow.js:1]: Looks like this js file doesn't have the latest updates in the main branch.

File: webapp/client/src/containers/_nav.js
Comment (yxu-lanl) [inline webapp/client/src/containers/_nav.js:1]: Looks like this js file doesn't have the latest updates in the main branch.

Comment (mflynn-lanl) [reply webapp/client/src/containers/_nav.js:1]: I updated to include the latest changes from the main branch

File: webapp/server/util/workflow.js
Comment (mflynn-lanl) [reply webapp/server/util/workflow.js:1]: I'm not sure what differences are from the latest updates to the main branch and which are mine

File: webapp/server/crons/workflowMonitor.js
Comment (mflynn-lanl) [reply webapp/server/crons/workflowMonitor.js:1]: I changed options path to be the same for all of the metaG workflows and updated workflowMonitor.js to use config/workflow.js

File: webapp/server/crons/pipelineMonitor.js
Comment (mflynn-lanl) [reply webapp/server/crons/pipelineMonitor.js:1]: I updated it to pipelineMonitor.js to use the value from the config file

File: webapp/server/util/workflow.js
Comment (yxu-lanl) [reply webapp/server/util/workflow.js:1]: https://github.com/microbiomedata/nmdc-edge/commit/69e0c5c23792321e4261ee602cf315505ed6f0cf
Screenshot 2024-05-15 at 10 31 53 AM

**Comment by mflynn-lanl** [comment]
I update my branch to include the changes from commit 69e0c5c
________________________________
From: yxu-lanl ***@***.***>
Sent: Wednesday, May 15, 2024 12:25 PM
To: microbiomedata/nmdc-edge ***@***.***>
Cc: Flynn, Mark C ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [microbiomedata/nmdc-edge] 42 sync assembly nmdc a (PR #159)


@yxu-lanl commented on this pull request.

________________________________

On webapp/server/util/workflow.js<https://urldefense.com/v3/__https://github.com/microbiomedata/nmdc-edge/pull/159*discussion_r1602076949__;Iw!!Bt8fGhp8LhKGRg!FJauLHlz-N8cIg7vQq3inJXtmRWlMH6QV6M4LRKDWfUqD_fvyJ4anIY3MJEfHJk0uphhv_ZH9QJatVI42Q4Q8iFw$>:

69e0c5c<https://urldefense.com/v3/__https://github.com/microbiomedata/nmdc-edge/commit/69e0c5c23792321e4261ee602cf315505ed6f0cf__;!!Bt8fGhp8LhKGRg!FJauLHlz-N8cIg7vQq3inJXtmRWlMH6QV6M4LRKDWfUqD_fvyJ4anIY3MJEfHJk0uphhv_ZH9QJatVI42YAzHiJP$>
Screenshot.2024-05-15.at.10.31.53.AM.png (view on web)<https://urldefense.com/v3/__https://github.com/microbiomedata/nmdc-edge/assets/38266469/e92a2b10-20b7-4386-8a3c-2c3ddaccba12__;!!Bt8fGhp8LhKGRg!FJauLHlz-N8cIg7vQq3inJXtmRWlMH6QV6M4LRKDWfUqD_fvyJ4anIY3MJEfHJk0uphhv_ZH9QJatVI42VjKT-Rh$>

**Review by yxu-lanl — APPROVED** [review]

**Pull request merged**